lib/kontrast/api_client.rb

```diff
@@ -0,0 +1,56 @@
+require 'faraday'
+
+module Kontrast
+    class ApiClient
+
+        attr_reader :responses, :env
+        attr_writer :headers
+
+        def initialize(env, host, app_id, app_secret, headers: {})
+            @env = env
+            @host = host
+            @app_id = app_id
+            @app_secret = app_secret
+            @connection = nil
+            @responses = {}
+            @headers = headers
+        end
+
+        def fetch(path, save_file: false, folder_name: "")
+            response = connection.get("#{path}") do |req|
```

Comment:
definitely, this is leftover from a time when the path was being concatenated here